torchvision/csrc/cuda/deform_conv2d_cuda.cu

```diff
@@ -133,7 +135,7 @@ __device__ scalar_t bilinear_interpolate(
 }
 
 template <typename scalar_t>
-__global__ void deformable_im2col_gpu_kernel(
```

Comment:
Removing "gpu" names from private methods. With namespaces we won't have conflicts any more.